src/types/publication.types.ts

```diff
@@ -99,7 +99,9 @@ export interface ReportPublication extends BasePublication {
   entityDescription: ReportEntityDescription;
 }
 
-export interface PublicationDate extends BackendType {
+export interface PublicationDateWithType extends BackendType, PublicationDate {}
+
+export interface PublicationDate {
```

Comment:
hvorfor er disse delt inn i to typer no? ville trodd at det alltid skal være et type felt der, evt at det er nullable

Reply:
vil gjenbruke PublicationDate på SearchResults, men der er det ikke type-felt.